src/CodeGenerator/src/Generator/OperationGenerator.php

```diff
@@ -123,22 +123,23 @@ public function generate(Operation $operation): void
 
     private function setMethodBody(Method $method, Operation $operation, ClassName $inputClass, ?ClassName $resultClass): void
     {
-        $params = ['$response', '$this->httpClient'];
         if ((null !== $pagination = $operation->getPagination()) && !empty($pagination->getOutputToken())) {
-            $params = \array_merge($params, ['$this', '$input']);
-        }
+            $body = '
+                $response = $this->getResponse($input = INPUT_CLASS::create($input)->request());
```

Comment:
We can make this a little bit more readable. 

```suggestion
                $input = INPUT_CLASS::create($input)->request()
                $response = $this->getResponse($input);
```

This will be the same as in the waiters.

Reply:
done